candies/cli/parsers/standard.py

```diff
@@ -140,8 +129,51 @@ def configure_flag(x: inspect.Parameter, config: Dict):
                              f"'{short}': "
                              f"it is not an alphabetic letter")
 
-        parser.add_argument(f'--{parameter.name}',
-                            f'-{short}',
+        return short
+
+    def prefixes_of(x: Union[Type, Any]) -> Optional[str]:
```

Comment:
The return type annotation should be `Optional[Tuple[str, str]]`.

Reply:
Changed to `Tuple[str, str]`. I omitted the `Optional`, because:
1) current implementation always returns some value;
2) I also feel like we should either return default prefixes `'-', '--'` or the specified ones. Not sure whether `argparse` allows to omit the prefixes at all.